<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgProjectSettings</class>
 <widget class="QDialog" name="DlgProjectSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="lblGameTitle">
       <property name="text">
        <string>Game Title:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>C++ Class Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtClassName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="txtTitleName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Data locations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLineEdit" name="txtAssetsLocation">
        <property name="text">
         <string>./</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="txtSourceLocation">
        <property name="text">
         <string>./</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnBrowseAssets">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="txtMetaDataLocation">
        <property name="text">
         <string>./</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btnBrowseSource">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnBrowseMetaData">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Game assets relative location:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Source code relative location:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Meta-data relative location:</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label_5</zorder>
     <zorder>txtAssetsLocation</zorder>
     <zorder>btnBrowseAssets</zorder>
     <zorder>label_6</zorder>
     <zorder>txtMetaDataLocation</zorder>
     <zorder>btnBrowseMetaData</zorder>
     <zorder>label_7</zorder>
     <zorder>txtSourceLocation</zorder>
     <zorder>btnBrowseSource</zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Initial Update FPS Cap (throttle)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbUpdateFpsCap">
       <property name="maximum">
        <number>300</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Total number of input mappings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbInputMaps"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Game Pixels per Meter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPixelsPerMeter"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkShowCursor">
     <property name="text">
      <string>Show Cursor in window</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgProjectSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgProjectSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
